site/docs/master/configuration.md

```diff
@@ -75,8 +75,8 @@ The timeout configuration block can be used to configure various timeouts for th
 
 | Field Name | Type| Default  | Description |
 |------------|-----|----------|-------------|
-| connection-idle-timeout| [duration][4] | `60s` | This field defines how long the proxy should wait while there are no active requests before terminating an HTTP connection. Set to 0 to disable the timeout. |
-| stream-idle-timeout| [duration][4] | `5m` | This field defines how long the proxy should wait while there is no stream activity before terminating a stream. Set to 0 to disable the timeout. |
+| connection-idle-timeout| [duration][4] | `60s` | This field defines how long the proxy should wait while there are no active requests (for HTTP/1.1) or streams (for HTTP/2) before terminating an HTTP connection. Set to 0 to disable the timeout. |
```

Comment:
Maybe even link back to the Envoy docs?

Another set of questions I've heard is what do these parameters apply to, http, tcp, websockets, etc. Would be super nice to clarify that as well.

Reply:
I added links, and I think the descriptions are now all pretty clear on these being HTTP timeouts, but let me know if not.

We could consider prefixing the actual field names with `http-` to be super-clear if that's something others think makes sense. Since HTTP is the main use case, I opted not to include it on these fields.